x/mongo/driver/session/session_pool.go

```diff
@@ -54,7 +61,10 @@ func NewPool(descChan <-chan description.Topology) *Pool {
 func (p *Pool) updateTimeout() {
 	select {
 	case newDesc := <-p.descChan:
-		p.timeout = newDesc.SessionTimeoutMinutes
+		p.latestTopology = topologyDescription{
+			kind:           newDesc.Kind,
+			timeoutMinutes: newDesc.SessionTimeoutMinutes,
+		}
```

Comment:
To check my understanding. For a load balanced deployment, this channel should receive only the first topology of kind `description.LoadBalanced` with `timeoutMinutes=0`, right?

Reply:
Yep, per the documentation in `topology.Topology#Subscribe`, the channel in the returned subscription is pre-populated with the current topology description, so that will be read here. There won't be any other descriptions coming on this channel because descriptions are only published when the topology is updated due to server monitoring or SRV polling routines, both of which are disabled in LB mode.